recipes/ogre/1.x/conanfile.py

```diff
@@ -0,0 +1,284 @@
+import os
+from conans import CMake, ConanFile, tools
```

Comment:
@uilianries, in reference to your [comment](https://github.com/conan-io/conan-center-index/pull/11102#issuecomment-1151040629) this line already imports `tools`.

Reply:
sorry, my mistake, the correct call is `tools.Version`, not `self.Version`